performance.lic

```diff
@@ -34,7 +32,7 @@ class Performance
       bput("stow my #{hand_armor}", 'You put', 'Stow what')
     end
 
-    return unless play_song
+    return unless play_song?(@settings, @song_list, is_instrument_worn, skip_clean)
```

Comment:
💡 I prefer to see this as `DRC.play_song?` (the same for the rest of the common functions)

Reply:
You're right, done and done.